### PR TITLE
Connect Vote to Agenda

### PR DESCRIPTION
**Pull request by davidwessman**
Adding some visuals to User views
Adding Agenda reference to Vote
Adding :selected to VotePost

### CODE REVIEW

File: app/views/admin/votes/show.html.erb [hunk 1]
```diff
@@ -1,39 +1,41 @@
...
     <table class="table">
       <% @vote.vote_options.with_deleted.order(count: :desc).each do |v| %>
         <tr>
           <td><b><%= vote_option_str(v) %></b></td>
-         <td><b><%= v.count %></b></td>
-       </tr>
+          <td><b><%= v.count %></b></td>
+        </tr>
       <% end %>
     </table>
```
Comment (navro) [inline app/views/admin/votes/show.html.erb]: In my version I added this to the end of this table:
```
     <tr>
        <td><i><%= "Σ röster / Σ blanka" %></i></td>
       <td><i><%= vote_stats(vote: @vote, adjusted: @adjusted) %></i></td>
     </tr>
```
and in votes_helper
```
  def vote_stats(vote, adjusted)
    pcount = vote.vote_posts.sum(:selected)
    ocount = vote.vote_options.sum(:count)

    if pcount == ocount
      pcount.to_s + ' / ' + (adjusted * vote.choices - pcount).to_s
    else
      'Something is wrong...'
    end
  end
```

File: app/controllers/admin/votes_controller.rb [hunk 2]
```diff
@@ -3,7 +3,7 @@ class Admin::VotesController < ApplicationController
   before_action :authorize
 
   def index
-    @votes_grid = initialize_grid(Vote)
+    @votes_grid = initialize_grid(Vote, include: :agenda, order: 'agendas.title')
```
Comment (navro) [inline app/controllers/admin/votes_controller.rb]: I think sort_index would be better :)

File: app/views/admin/votes/index.html.erb [hunk 3]
```diff
@@ -10,11 +12,17 @@
     <%= link_to t('agenda.manage'), admin_agendas_path, class: 'btn secondary' %>
   <% end %>
 </div>
+
 <div class="col-md-12">
   <%= grid(@votes_grid) do |grid|
     grid.column name: t('attributes.name'), attribute: 'title' do |v|
       link_to v.title, admin_vote_path(v)
     end
+    grid.column name: model_name(Agenda), assoc: :agenda, attribute: 'title' do |v|
```
Comment (navro) [inline app/views/admin/votes/index.html.erb]: sort_index

File: app/assets/stylesheets/partials/_votes.scss [hunk 4]
```diff
@@ -35,4 +35,31 @@
...
+    text-align: center;
+
+    i {
+      color: $fsek-orange;
+      font-size: 20px;
+    }
+  }
+
+  .not-present {
+    font-size: 28px;
```
Comment (navro) [inline app/assets/stylesheets/partials/_votes.scss]: This font size is too large (almost as large as h1)

File: app/assets/stylesheets/partials/_votes.scss [hunk 5]
```diff
@@ -35,4 +35,31 @@
...
+
+    i {
+      color: $unify-red;
+      font-size: 35px;
+    }
+  }
+
+  .present {
+    font-size: 28px;
```
Comment (navro) [inline app/assets/stylesheets/partials/_votes.scss]: Font size

File: app/views/admin/votes/_form.html.erb [hunk 6]
```diff
@@ -1,5 +1,6 @@
 <%= simple_form_for [:admin, @vote] do |f| %>
   <%= f.input :title %>
+  <%= f.association :agenda, collection: Agenda.all, label_method: :to_s  %>
```
Comment (navro) [inline app/views/admin/votes/_form.html.erb]: Order by sort_index

File: app/views/votes/index.html.erb [hunk 7]
```diff
@@ -1,13 +1,39 @@
...
+
+  <div class="headline">
+    <h3><%= t('agenda.current') %></h3>
+  </div>
+
+  <% if @vote_status.agenda.present? %>
+    <span>
+      <%= @vote_status.agenda.to_s %><br>
+    </span>
+
```
Comment (navro) [inline app/views/votes/index.html.erb]: A line break after this would be nice :)

**Comment by navro** [comment]
Just some small comments

**Comment by davidwessman** [comment]
@navro Fixed said comments 😄

**Comment by navro** [comment]
> @navro Fixed said comments :smile:

Merge! :smile:

**Pull request merged**